Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationPponSearch.cshtml

```diff
@@ -71,6 +71,12 @@
         </div>
     </div>
 
+
+    @if (!string.IsNullOrEmpty(Model.FeedbackMessage))
+    {
+        <p class="govuk-body">@Html.Raw(Model.FeedbackMessage)</p>
```

Comment:
H1 inside a p is invalid markup, you should remove this element and call Html.Raw directly